conda/version.py

```diff
@@ -163,7 +163,7 @@ def __init__(self, version):
         version = version[-1].split('+')
         if len(version) == 1:
             # no local version
-            self.local = ['0']
+            self.local = []
```

Comment:
This change allows me to differentiate the case where a local version was not supplied from the case where the local version is a simple zero. I can't see anywhere in the code where an assumption of a non-empty local version is needed.